algrtm/next.config.js

```diff
@@ -0,0 +1,18 @@
+/** @type {import('next').NextConfig} */
+const nextConfig = {
+  output: 'export',
+
+  basePath: '/algrtm',
+  assetPrefix: '/algrtm',
```

Comment:
I'm not quite sure if these two lines: 
```js
basePath:"/algrtm",
assetPrefix: "/algrtm",
```
are needed for deployment of the project, or do you just want the path to always contain "/algrtm" after "algrtm.com" as in the picture below:

![image](https://github.com/Nospoko/algrtm/assets/26410548/2281e322-550d-4a60-a05b-3a4b774fb5e9)

Using them locally, in order for the page not to return a 404 error, we need to include "/algrtm" to our url (http://localhost:3001/algrtm), and in order for the assets to be displayed correctly, we also need to use the "/algrtm" prefix, so in the Header.js file, make a change like this in the place where `ellipse.svg` is used:
```js
<Image
src='/algrtm/ellipse.svg'
width={24}
height={24}
alt='ellipse icon'
/>
```

Alternatively, after removing these lines:
```js
basePath:"/algrtm",
assetPrefix:"/algrtm",
```
everything should work like this:
![image](https://github.com/Nospoko/algrtm/assets/26410548/ecdbfeaa-b3f1-4cff-bc25-7313314834fe)